packages/clerk-js/src/ui/contexts/ClerkUIComponentsContext.tsx

```diff
@@ -101,6 +116,18 @@ export const useSignInContext = (): SignInContextType => {
   const { queryParams } = useRouter();
   const options = useOptions();
 
+  const initialValuesFromQueryParams = useMemo(() => {
+    const props: Record<string, string> = {};
+    const searchParams = new URLSearchParams(window.location.search);
```

Comment:
❓ Same as above about accessing `window`

Reply:
I've replaced it with `router.queryString` anyway.